MANIFEST.in

```diff
@@ -1 +1,11 @@
-include AUTHORS COPYING README.md
```

Comment:
Seems like ``pbr`` doesn't support it - according to the docs it will attempt to auto-generate the list, filtering .gitignore files, but it doesn't allow custom filtering
https://docs.openstack.org/developer/pbr/#manifest

So we probably still need `MANIFEST.in`.